lib/coinbase/address/wallet_address.rb

```diff
@@ -81,9 +81,6 @@ def trade(amount, from_asset_id, to_asset_id)
         wallet_id: wallet_id
       )
 
-      # If a server signer is managing keys, it will sign and broadcast the underlying trade transaction out of band.
```

Comment:
is this related to this PR ?

Reply:
Nah this is just a git relic, you can ignore.